demisto_sdk/commands/common/content/objects_factory.py

```diff
@@ -1,10 +1,11 @@
+from demisto_sdk.commands.common.content.objects.pack_objects.pre_preocess_rules.pre_process_rules import PreProcessRules
```

Comment:
remove, this was automatically added by vscode

Reply:
https://github.com/demisto/demisto-sdk/commit/7939f80a0f2adb1a509f695135d38738e35fff6a